<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMainWindow</class>
 <widget class="QMainWindow" name="MyMainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>1128</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>文件内容修改工具</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Graphics.ico</normaloff>Graphics.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1600</width>
     <height>1000</height>
    </size>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>49</y>
      <width>1590</width>
      <height>880</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1590</width>
      <height>880</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>841</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>301</width>
       <height>820</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Microsoft JhengHei</family>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>模块列表</string>
     </property>
     <widget class="QPushButton" name="moduleButton1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>271</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>模块1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="moduleButton2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>271</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>模块2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="moduleButton3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>271</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>模块3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="moduleButton4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>271</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>模块4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="moduleButton5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>271</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>模块5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="moduleButton6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>530</y>
        <width>271</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>模块6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="moduleButton7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>630</y>
        <width>271</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>模块7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="moduleButton8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>730</y>
        <width>271</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>模块8</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>1241</width>
       <height>840</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>900</width>
       <height>840</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Microsoft JhengHei</family>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="title">
      <string>文本内容</string>
     </property>
     <widget class="QTextEdit" name="infoTextEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>1021</width>
        <height>771</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="saveButton">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>50</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cancelButton">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>150</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="moduleGroupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>1051</width>
        <height>811</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei</family>
        <pointsize>6</pointsize>
       </font>
      </property>
      <property name="title">
       <string>模块内容</string>
      </property>
     </widget>
     <widget class="QPushButton" name="historyButton">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>720</y>
        <width>151</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>历史记录</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="historyTextEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>1021</width>
        <height>771</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="okButton">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>620</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>OK√</string>
      </property>
     </widget>
     <zorder>moduleGroupBox</zorder>
     <zorder>infoTextEdit</zorder>
     <zorder>saveButton</zorder>
     <zorder>cancelButton</zorder>
     <zorder>historyButton</zorder>
     <zorder>historyTextEdit</zorder>
     <zorder>okButton</zorder>
    </widget>
   </widget>
   <widget class="QLabel" name="pathLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>15</y>
      <width>1041</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fileSrcLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei</family>
     </font>
    </property>
    <property name="text">
     <string>文件路径：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="pOpen"/>
   <addaction name="pAbout"/>
   <addaction name="pChart"/>
   <addaction name="pCopyRight"/>
   <addaction name="pSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="pOpen">
   <property name="icon">
    <iconset>
     <normaloff>../03_QMainWindow/open.png</normaloff>../03_QMainWindow/open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="pAbout">
   <property name="icon">
    <iconset>
     <normaloff>../03_QMainWindow/about.png</normaloff>../03_QMainWindow/about.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
   <property name="toolTip">
    <string>关于App</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="pChart">
   <property name="icon">
    <iconset>
     <normaloff>../03_QMainWindow/chart.png</normaloff>../03_QMainWindow/chart.png</iconset>
   </property>
   <property name="text">
    <string>图表</string>
   </property>
   <property name="toolTip">
    <string>图表</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="pCopyRight">
   <property name="icon">
    <iconset>
     <normaloff>../03_QMainWindow/copyRight.png</normaloff>../03_QMainWindow/copyRight.png</iconset>
   </property>
   <property name="text">
    <string>版权</string>
   </property>
   <property name="toolTip">
    <string>版权</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="pSetting">
   <property name="icon">
    <iconset>
     <normaloff>../03_QMainWindow/setting.png</normaloff>../03_QMainWindow/setting.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="toolTip">
    <string>设置</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
